alarm-frontend/src/App.js

```diff
@@ -1,22 +1,43 @@
-import AddAlarm from "./Components/AddAlarm";
+import React, { useState } from "react";
 import Clock from "./Components/Clock";
 import ContextAlarm from "./Components/Context";
 import AlarmList from "./Components/AlarmList";
+import Math from "./Components/Math";
 
 function App() {
-  return (
-    <section>
-      <div>
+  const [type, setCurrentType] = useState("Base");
+  const [difficulty, setCurrentDiff] = useState();
+
+  if ( type === "Base") {
+    return (
+        <section>
+          <div>
+            <div>
+              <ContextAlarm>
+                <Clock />
+                <AlarmList 
+                setCurrentType={setCurrentType}
+                setCurrentDiff = {setCurrentDiff}
+                />
+              </ContextAlarm>
+            </div>
+          </div>
+        </section>
+      );
+  } else if (type === "Math") {
```

Comment:
Since the other if return, no need for else here
```suggestion
  } if (type === "Math") {
```